tests/test_bot.py

```diff
@@ -1910,7 +1910,7 @@ async def test_set_webhook_delete_webhook_drop_pending_updates(
         self, bot, drop_pending_updates, monkeypatch
     ):
         async def make_assertion(url, request_data: RequestData, *args, **kwargs):
-            data = request_data.json_parameters
+            data = request_data.parameters
```

Comment:
why? 🤔

Reply:
Sorry, I don't quite follow. Are we talking about a bug in `_bot.py` or a bug in the test?

Reply:
both actually. point 1. is about the bot method, 2. is this test

Reply:
> do we not want to pass False values to Telegram?

I guess we want to and hopefully we had `if value is not None` checks instead in those cases …

so the issue was that `request_data.json_parameters["drop_pending_updates"]` was a string, right? and `request_data.parameters["drop_pending_updates"]` is a bool or `None`? can't we drop the `bool(…)` in line 1914, then?

Reply:
> so the issue was that `request_data.json_parameters["drop_pending_updates"]` was a string, right? 

yes

> and `request_data.parameters["drop_pending_updates"]` is a bool or `None`? can't we drop the `bool(…)` in line 1914, then?

yes